src/ScnSocialAuth/Authentication/Adapter/HybridAuth.php

```diff
@@ -319,6 +319,12 @@ protected function instantiateLocalUser()
     protected function facebookToLocalUser($userProfile)
     {
         if (!isset($userProfile->emailVerified)) {
+            //some facebook user's dont have email address
+            if ($userProfile->email == ''){
+              $userProfile->email = $userProfile->identifier.'@facebook.com';
+              $userProfile->email = $userProfile->emailVerified;
```

Comment:
Why set `$userProfile->email` in each of these lines? Also, you are setting it to `$userProfile->emailVerified` which we just confirmed wasn't set.